tools/c7n_oci/c7n_oci/resources/compute.py

```diff
@@ -0,0 +1,280 @@
+    https://docs.oracle.com/en-us/iaas/tools/python/latest/api/core/client/oci.core.ComputeClient.html#oci.core.ComputeClient.update_instance
+
+    .. code-block:: yaml
+
+        policies:
+            - name: perform-update-instance-action
+              resource: oci.instance
+              actions:
+                - type: update-instance
+
+    """  # noqa
+
+    schema = type_schema(
+        "update-instance", params={"type": "object"}, rinherit=OCIBaseAction.schema
+    )
+
+    def perform_action(self, resource):
+        client = self.manager.get_client()
+        params_dict = {}
+        params_model = {}
+        if self.data.get("params") and self.data.get("params").get("instance_id"):
+            params_dict["instance_id"] = self.data.get("params").get("instance_id")
+        else:
+            params_dict["instance_id"] = resource.get("id")
+        if self.data.get("params").get("update_instance_details"):
+            update_instance_details_user = self.data.get("params").get("update_instance_details")
+            params_model = self.update_params(resource, update_instance_details_user)
+            params_dict["update_instance_details"] = oci.core.models.UpdateInstanceDetails(
+                **params_model
+            )
+        response = client.update_instance(
+            instance_id=params_dict["instance_id"],
+            update_instance_details=params_dict["update_instance_details"],
+        )
+        log.info(f"Received status {response.status} for PUT:update_instance {response.request_id}")
+        return response
+
+
+@Instance.action_registry.register("remove-tag")
+class RemoveTagActionInstance(RemoveTagBaseAction):
+    """
+    Remove Tag Action
+
+    :example:
+
+        Remove the specified tags from the resource. Defined tag needs to be referred as 'namespace.tagName' as below in the policy file.
+
+    .. code-block:: yaml
+
+        policies:
+            - name: remove-tag
+              resource: oci.instance
+            actions:
+              - type: remove-tag
+                defined_tags: ['cloud_custodian.environment']
+                freeform_tags: ['organization', 'team']
+
+    """  # noqa
+
+    def perform_action(self, resource):
+        client = self.manager.get_client()
+        params_dict = {}
+        params_dict["instance_id"] = resource.get("id")
+        original_tag_count = self.tag_count(resource)
+        params_model = self.remove_tag(resource)
+        updated_tag_count = self.tag_count(params_model)
+        params_dict["update_instance_details"] = oci.core.models.UpdateInstanceDetails(
+            **params_model
+        )
+        if self.tag_removed_from_resource(original_tag_count, updated_tag_count):
+            response = client.update_instance(
+                instance_id=params_dict["instance_id"],
+                update_instance_details=params_dict["update_instance_details"],
+            )
+            log.info(
```

Comment:
most of the remove-tag actions are doing log.info per resource, this should probably be log.debug as its pretty verbose with any cardinality of resources.

Reply:
Changed from info to debug, will be available in next push